app/helpers/course/statistics_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module Course::StatisticsHelper
+  # Convert time in seconds to HH:MM:SS format.
+  def seconds_to_str(total_seconds)
```

Comment:
I have considered that, but think this helper is quite specific to staff summary though ( I cannot find other use cases in somewhere else). Other places maybe we can use rails default helper like `time_in_words`.